src/Symfony/Bridge/Doctrine/Tests/Security/User/EntityUserProviderTest.php

```diff
@@ -59,9 +61,7 @@ public function testLoadUserByUsernameWithUserLoaderRepositoryAndWithoutProperty
     {
         $user = new User(1, 1, 'user1');
 
-        $repository = $this->getMockBuilder('Symfony\Bridge\Doctrine\Security\User\UserLoaderInterface')
```

Comment:
changes on this file should be reverted (and/or be submitted on 3.4)

Reply:
> and/or be submitted on 3.4

My failed attempt to do so is documented as #33771. 🙈 

The changes are necessary because the mocks are currently incomplete and because of that they wouldn't pass a return type declaration I've added.